### PR TITLE
2.0.2 mainnet docs

### PR DESCRIPTION
**Pull request by badrogger**

### CODE REVIEW

**Review by gannakulikova — APPROVED** [review]

**Review by dmytrotkk — APPROVED** [review]

File: docs/modules/ROOT/pages/parameters.adoc [hunk 1]
```diff
@@ -25,25 +25,25 @@
 :docker-config: live-restore enabled https://docs.docker.com/config/containers/live-restore/[docker docs]
 
 // skale software
-:node-cli: https://github.com/skalenetwork/node-cli/releases/download/2.0.1/skale-2.0.1-Linux-x86_64[2.0.1]
+:node-cli: https://github.com/skalenetwork/node-cli/releases/download/2.0.2/skale-2.0.1-Linux-x86_64[2.0.2]
```
Comment (cstrangedk) [inline docs/modules/ROOT/pages/parameters.adoc]: please double check version

File: docs/modules/ROOT/pages/parameters.adoc [hunk 2]
```diff
@@ -25,25 +25,25 @@
...
-:skale-node: https://github.com/skalenetwork/skale-node/releases/tag/2.0.0[2.0.0]
-:skaled: https://github.com/skalenetwork/skaled/releases/tag/3.7.3-stable.0[3.7.3-stable.0]
+:skale-node: https://github.com/skalenetwork/skale-node/releases/tag/2.0.2[2.0.2]
+:skaled: https://github.com/skalenetwork/skaled/releases/tag/3.7.5-stable.0[3.7.5-stable.0]
 :docker-lvmpy: https://github.com/skalenetwork/docker-lvmpy/releases/tag/1.0.1-stable.3[1.0.1-stable.3]
-:transaction-manager: https://github.com/skalenetwork/transaction-manager/releases/tag/2.0.1[2.0.1]
-:skale-admin: https://github.com/skalenetwork/skale-admin/releases/tag/2.0.1[2.0.1]
-:bounty-agent: https://github.com/skalenetwork/bounty-agent/releases/tag/2.0.1-stable.0[2.0.1-stable.0]
-:skale-watchdog: https://github.com/skalenetwork/skale-watchdog/releases/tag/2.0.0-stable.0[2.0.0-stable.0]
+:transaction-manager: https://github.com/skalenetwork/transaction-manager/releases/tag/2.0.1[2.0.2]
```
Comment (cstrangedk) [inline docs/modules/ROOT/pages/parameters.adoc]: double check version tag

File: docs/modules/ROOT/pages/parameters.adoc [hunk 3]
```diff
@@ -25,25 +25,25 @@
...
+:skale-admin: https://github.com/skalenetwork/skale-admin/releases/tag/2.0.1[2.0.2]
```
Comment (cstrangedk) [inline docs/modules/ROOT/pages/parameters.adoc]: ibid.

File: docs/modules/ROOT/pages/parameters.adoc [hunk 4]
```diff
@@ -25,25 +25,25 @@
...
+:bounty-agent: https://github.com/skalenetwork/bounty-agent/releases/tag/2.0.1-stable.0[2.0.2-stable.0]
```
Comment (cstrangedk) [inline docs/modules/ROOT/pages/parameters.adoc]: ibid.

File: docs/modules/ROOT/pages/parameters.adoc [hunk 5]
```diff
@@ -25,25 +25,25 @@
...
+:skale-watchdog: https://github.com/skalenetwork/skale-watchdog/releases/tag/2.0.0-stable.0[2.0.2-stable.0]
```
Comment (cstrangedk) [inline docs/modules/ROOT/pages/parameters.adoc]: ibid.

File: docs/modules/ROOT/pages/soft-upgrade.adoc [hunk 6]
```diff
@@ -1,47 +1,21 @@
...
 :sectnums:
 :sectnumlevels: 1
 
-For SKALE Validator CLI initialization use SKALE Manager ABI:
+[WARNING]
+Be sure to follow specific upgrade schedule assigned to your node(s). Follow guidance from the core team.
 
-`{MANAGER_CONTRACTS_ABI_URL}`
+[NOTE]
+From the `2.0.2` SKALE node release docker is listening on additional socket which is located in `/var/run/skale/docker.sock`.
```
Comment (cstrangedk) [inline docs/modules/ROOT/pages/soft-upgrade.adoc]: >>
With the `2.0.2` version, please note that the SKALE node docker now listens on an additional socket which is located in `/var/run/skale/docker.sock`. 
A new socket path will be added to `daemon.json` config during update. In addition, the service configuration for docker daemon will be changed to ensure that `/var/run/skale` is created.

**Review by cstrangedk — CHANGES_REQUESTED** [review]

**Review by dmytrotkk — APPROVED** [review]

**Review by cstrangedk — APPROVED** [review]

**Pull request merged**